src/Features/CSharp/Portable/Completion/CompletionProviders/OperatorsAndIndexer/OperatorIndexerCompletionProviderBase.cs

```diff
@@ -31,7 +31,13 @@ internal abstract class OperatorIndexerCompletionProviderBase : LSPCompletionPro
 
         protected abstract int SortingGroupIndex { get; } // Indexer, operators and conversion should be listed grouped together.
 
-        protected abstract ImmutableArray<CompletionItem> GetCompletionItemsForTypeSymbol(ITypeSymbol container, bool isAccessedByConditionalAccess, SemanticModel semanticModel, int position);
+        protected abstract ImmutableArray<CompletionItem> GetCompletionItemsForTypeSymbol(
+            ITypeSymbol container,
+            bool isAccessedByConditionalAccess,
+            ExpressionSyntax expression,
+            SemanticModel semanticModel,
+            int position,
+            CancellationToken cancellationToken);
```

Comment:
in general, order from widest to narrowest (with cancellation always last).  so here, that would be: